src/frontend/admin/src/components/testing/presentational/hook-form/RHFValuesChange/RHFValuesChange.spec.e2e.tsx

```diff
@@ -0,0 +1,158 @@
+          }}
+        />
+      </PlaywrightCustomRouter>,
+      { hooksConfig: { customRouting: true } },
+    );
+    expect(values).toEqual(null);
+    await component.getByLabel("Name", { exact: true }).fill("Doe");
+    await component.getByLabel("Select", { exact: true }).click();
+    await page.getByRole("option", { name: "First" }).click();
+    await component
+      .getByTestId("radio-input-enable")
+      .getByLabel("True")
+      .click();
+
+    await delay(50);
+    expect(routerHasChange).toEqual(true);
+    expect(router.asPath).toEqual("/?name=Doe&select=1&enable=true");
+    await component
+      .getByTestId("radio-input-enable")
+      .getByLabel("None")
+      .click();
+    await delay(25);
+    expect(router.asPath).toEqual("/?name=Doe&select=1");
+  });
+
+  test("Check that the submit function is called at the right time without form validation", async ({
+    mount,
+  }) => {
+    const router = mockRouter;
+    let values = null;
+    const component = await mount<HooksConfig>(
+      <PlaywrightCustomRouter router={router}>
+        <RHFValuesChangeTestWrapper
+          debounceTime={200}
+          onSubmit={(newValues) => {
+            values = newValues;
+          }}
+        />
+      </PlaywrightCustomRouter>,
+      { hooksConfig: { customRouting: true } },
+    );
+    expect(values).toEqual(null);
+    await component.getByLabel("Name", { exact: true }).fill("Doe");
+    await component.click();
+    await component.getByLabel("Name", { exact: true }).fill("");
+    expect(values).toEqual(null);
+    await delay(220);
+    await component.getByLabel("Name", { exact: true }).fill("John");
+    await delay(220);
+    expect(values).toEqual({ enable: "", name: "John", select: "None" });
+  });
+
+  test("Check that the submit function is called at the right time with form validation", async ({
+    mount,
+    page,
+  }) => {
+    const router = mockRouter;
+    let values = null;
+    const component = await mount<HooksConfig>(
+      <PlaywrightCustomRouter router={router}>
+        <RHFValuesChangeTestWrapper
+          debounceTime={100}
+          enableSchema={true}
+          onSubmit={(newValues) => {
+            values = newValues;
+          }}
+        />
+      </PlaywrightCustomRouter>,
+      { hooksConfig: { customRouting: true } },
+    );
+    expect(values).toEqual(null);
+    await component.getByLabel("Select", { exact: true }).click();
+    await page.getByRole("option", { name: "First" }).click();
+    await delay(120);
+    expect(values).toEqual(null);
+    await component
+      .getByTestId("radio-input-enable")
+      .getByLabel("True")
+      .click();
+    await delay(120);
+    expect(values).toEqual(null);
+    await component.getByLabel("Name", { exact: true }).fill("Doe");
+    await delay(120);
+    expect(values).toEqual({ enable: "true", name: "Doe", select: "1" });
+  });
+
+  test("Check url when form change with formValuesToFiltersValues props", async ({
```

Comment:
It seems that do not test refreshing the page, in order to make sure the chips are loaded correctly with the according metadata

Reply:
It's done here : 

https://github.com/openfun/joanie/pull/688/commits/70bb50c3eb04519ef52cd778146906563c08da00#diff-6e2a9f52254714a38d0010b7841ed92b7aa52cd6d57a809ce352ebfba394ac24R141-R143